libs/blocks/tabs/tabs.css

```diff
@@ -492,6 +498,77 @@
   text-shadow: revert;
 }
 
+/* Segmented Control Variant */
+.tabs.segmented-control .tabList {
+  background: none;
+  box-shadow: none;
+  display: flex;
+  align-items: center;
+  position: relative;
+  border-radius: var(--m-rounded-corners);
+  overflow: hidden;
+}
+
+.tabs.segmented-control .tabList .tab-list-container {
+  background: #f8f8f8;
+  overflow: hidden;
+  border-radius: var(--m-rounded-corners);
+  padding: 1px;
+  min-width: 298px;
+}
+
+.tabs.segmented-control .tabList button {
+  padding: 10px;
+  border-radius: var(--m-rounded-corners);
+  border: none;
+  font-weight: 700;
+  color: var(--tabs-pill-bg-color);
+  font-size: 14px;
+  flex: 1 0 auto;
+}
+
+.tabs.segmented-control .tabList button[aria-selected="true"],
+.tabs.segmented-control .tabList button[aria-checked="true"] {
+  background: var(--tabs-pill-bg-color);
+  color: var(--color-white);
+}
+
+.tabs.segmented-control .tabList button.paddle {
+  position: absolute;
+  border-bottom: none;
+  width: 54px;
+  height: 40px;
+  box-shadow: none;
+  flex-shrink: 0;
+  justify-content: end;
+  padding: 0 !important;
+  z-index: 4;
+}
+
+.tabs.segmented-control .tabList button.paddle-left { 
+  background: linear-gradient(270deg, rgba(255, 255, 255, 0.00) 28.48%, #FFF 50.3%);
+}
+
+.tabs.segmented-control .tabList button.paddle-right { 
+  background: linear-gradient(270deg, #FFF 49.7%, rgba(255, 255, 255, 0.00) 71.52%);
+}
+
+.tabs.segmented-control .paddle svg {
+  margin: unset;
+  width: 24px;
+  height: 24px;
+  color: #505050;
+}
+
+.tabs.segmented-control .tabList button.paddle:disabled {
+  background: none;
+  display: none;
+}
+
+.tabs.segmented-control .paddle:disabled svg {
+  opacity: 0;
+}
```

Comment:
This is already defined for `.tabs .paddle:disabled svg`, why is the repetition needed?